README.md

```diff
@@ -14,367 +14,11 @@ level of backwards compatibility to the official releases.
 
 ---
 
-## Table of contents
+## Documentation
 
-- [Releases and Versioning](#releases-and-versioning)
-  - [Currently Maintained Versions](#currently-maintained-versions)
-- [Requirements](#requirements)
-- [Installation](#installation)
-  - [Manual Install](#manual-install)
-  - [Composer](#composer)
-  - [Git](#git)
-- [Secure your installation](#secure-your-installation)
-  - [Apache .htaccess](#apache-htaccess)
-  - [Nginx](#nginx)
-- [Magento 1 Compatibility](#magento-1-compatibility)
-- [Changes](#changes)
-  - [Between Magento 1.9.4.5 and OpenMage 19.x](#between-magento-1945-and-openmage-19x)
-  - [Since OpenMage 19.5.0 / 20.1.0](#since-openmage-1950--2010)
-  - [New Config Options](#new-config-options)
-  - [New Events](#new-events)
-  - [Changes to SOAP/WSDL](#changes-to-soapwsdl)
-- [Development Environment with ddev](#development-environment-with-ddev)
-- [PhpStorm Factory Helper](#phpstorm-factory-helper)
-- [PhpStorm File-Watcher for SCSS files](#phpstorm-file-watcher-for-scss-files)
-- [Public Communication](#public-communication)
-- [Maintainers](#maintainers)
-- [License](#license)
-- [Contributors](#contributors-)
+See: https://OpenMage.github.io/magento-lts/
```

Comment:
```suggestion
See: https://docs.openmage.org/
```

Reply:
My settings ...

![ghp](https://github.com/user-attachments/assets/c57688ce-e069-47e6-be24-cb0c997fd3a8)

As long as automatic deployment works docs.openmage.org is even better. Thanks